media/js/firefox/lockwise/lockwise.js

```diff
@@ -0,0 +1,27 @@
+(function() {
+    'use strict';
+
+    var client = window.Mozilla.Client;
+    var version = client._getFirefoxMajorVersion();
+
+    if (client.isMobile) {
+        document.querySelector('.for-firefox-70-and-above').remove();
```

Comment:
This isn't a blocker, and something we could always fix up later - but we may want to consider the edge case of this JS either not running (i.e. in a legacy browser, or a noscript extension), or failing for some unforseen reason. Until this code runs, all the CTA's are visible on the page:

![image](https://user-images.githubusercontent.com/400117/66300067-90c0ef80-e8ec-11e9-9672-59f1f857a046.png)

An alternative way to approach this might be to hide all of these JS dependent CTA's by default (using CSS), and then to only show them explicitly here by using a CSS class hook.

Reply:
I have given them all the "hidden" class and remove the class from the button via javascript, if the script doesn't run the buttons are hidden.